README.md

```diff
@@ -136,6 +136,84 @@ custom:
   datadog:
     enabled: ${self:custom.staged.dd_enabled, true}
 ```
+### Serverless Monitors 
+
+Serverless has seven recommended monitors with default values pre-configured. 
+
+| Monitor              | Metrics                                                                                   | Default Threshold | Serverless Monitor ID |
+|:--------------------:|:-----------------------------------------------------------------------------------------:|:-----------------:|:---------------------:|
+| High Error Rate      | `aws.lambda.errors`/`aws.lambda.invocations`                                              | >= 10%            | `high_error_rate`     |
+| Timeout              | `aws.lambda.duration.max`/`aws.lambda.timeout`                                            | >= 1              | `timeout`             |
+| Out of Memory        | `aws.lambda.lambda.enhanced.max_memory_used`/<br>`aws.lambda.memorysize`                  | >= 1              | `out_of_memory`       |
+| High Iterator Age    | `aws.lambda.iterator_age.maximum`                                                         | >= 24 hrs         | `high_iterator_age`   |
+| High Cold Start Rate | `aws.lambda.enhanced.invaocations(cold_start:true)`/<br>`aws.lambda.enhanced.invocations` | >= 20%            | `high_cold_start_rate`|
+| High Throttles       | `aws.lambda.throttles`/`aws.lambda.invocations`                                           | >= 20%            | `high_throttles`      |
+| Increased Cost       | `aws.lambda.enhanced.estimated_cost`                                                      | &#8593;20%        | `increased_cost`      |
+ 
+#### To Enable and Configure a Recommended Serverless Monitor 
+
+To create a serverless monitor, you must use its respective  serverless monitor ID in addition to passing in the API key and Application key in the `serverless.yml` file. If you’d like to further configure the specific parameter values for a recommended monitor, you can directly define parameters below the serverless monitor ID. Parameters not specified under a recommended monitor will use the default recommended value. For further documentation on monitor parameters, see the [Datadog Monitors API](https://docs.datadoghq.com/api/latest/monitors/#create-a-monitor). You can directly update a monitor by changing the parameter values. To delete a monitor, remove the monitor from the `serverless.yml` template. 
+
+##### To create a recommended monitor with the default values
+Define the appropriate serverless monitor ID without specifying any parameter values
+
+```
+custom:
+ datadog:
+   addLayers: true
+   monitorsApiKey: ${file(./config.json):monitorsApiKey}
+   monitorsAppKey: ${file(./config.json):monitorsAppKey}
+   monitors:
+     - high_error_rate:
```

Comment:
Is there supposed to be a `:` at the end of this line?

Reply:
Yup, I believe that's part of the serverless.yml syntax so I'm not sure if I can do anything about it, although I do agree it looks a little off.